src/platform/site-wide/sass/formation-overrides/_functions.scss

```diff
@@ -0,0 +1,16 @@
+
+@function is-rem($value) {
+  @if unit($value) == "rem" {
+    @return true;
+  }
+}
+
+// TODO: Instead of converting to px, we should calculate the existing rem
+// based on the USWDS 16px base and return it again as rem.
+@function uswds-override($original) {
+  @if is-rem($original) {
+    $rem-to-px: ($original / 1rem) * 16;
+    @return ($rem-to-px) + "px";
```

Comment:
This function is [essentially the same thing as what we're doing in the component-library ](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/global/_functions.scss)to convert USWDS `rem` to `px`. We are doing that to avoid having to deal with the base font size in the components.

But we will actually want this function to convert the existing `rem` to another `rem` value based on the USWDS 16px base font. We need this to be rem still for a number of reasons. First, a11y concerns but also there are other stylesheets that seem to require a rem value for calculations (like the `accordions` import that I had to comment out in the minimal stylesheet)